pkg/api/controllers/fileops.go

```diff
@@ -0,0 +1,204 @@
+// Copyright (c) 2019 OpenSDS Authors All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
+This module implements a entry into the OpenSDS northbound service to upload and download configuration files
+
+*/
+
+package controllers
+
+import (
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+
+	log "github.com/golang/glog"
+
+	"github.com/opensds/opensds/pkg/controller/client"
+	"github.com/opensds/opensds/pkg/model"
+)
+
+// prometheus constants
+var PROMETHEUS_CONF_HOME string
```

Comment:
Please define the variable in this format like `PrometheusCONFHome`, also this style would be better:
```go
var (
    PrometheusCONFHome string
    PrometheusURL string
)
```

Reply:
refactored as per IDE suggestion, to following
var PrometheusConfHome string
var PrometheusUrl string
var PrometheusConfFile string

// alert manager constants
var AlertmgrConfHome string
var AlertmgrUrl string
var AlertmgrConfFile string

var GrafanaConfHome string
var GrafanaRestartCmd string
var GrafanaConfFile string

var ReloadPath string
var BackupExtension string